### PR TITLE
CT-323 Add Open/Close case tabs

### PR DESCRIPTION
**Pull request by aliuk2012**
https://dsdmoj.atlassian.net/browse/CT-323

- Addes "Open/Closed" case tabs on the case list view

Still to do:
- Create "Open/Closed" scope and display filtered results on the page

### CODE REVIEW

File: spec/features/case_list_view_spec.rb [hunk 1]
```diff
@@ -51,4 +51,27 @@
...
+    visit '/'
+    within('.section-tabs .active') do
+      expect(page).to have_content('Open cases')
+    end
+  end
+
+  scenario '"Open cases" tab should be selected if clicked' do
+    visit '/'
+    click_link 'Open cases'
+    within('.section-tabs .active') do
```
Comment (misaka) [inline spec/features/case_list_view_spec.rb]: Does it make sense to put this kind of thing into SitePrisms?

**Comment by misaka** [comment]
:100:

File: spec/features/case_list_view_spec.rb (hunk 1 above)
Comment (misaka) [inline spec/features/case_list_view_spec.rb]: @aliuk2012 Just wondering the above ^^^ should we bother with SitePrisms?

Comment (aliuk2012) [inline spec/features/case_list_view_spec.rb]: I've refactored the tests to use SitePrism

**Comment by aliuk2012** [comment]
#204 closes this PR

**Pull request closed without merging**